examples/file-uploader/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "file-uploader-example",
+  "version": "0.0.1",
+  "description": "File uploader example",
+  "main": "index.js",
+  "scripts": {
+    "pretest": "browserify js/test.js --debug -t /usr/lib/node_modules/babelify --outfile js/test_build.js",
+    "test": "node js/test_build.js | ./node_modules/tap-spec/bin/cmd.js",
+    "build": "browserify js/main.js --debug -t /usr/lib/node_modules/babelify --outfile js/build.js"
```

Comment:
Those paths does not work for me. I don't think you need qualified filenames when commands are being run through npm. I think npm sets up the correct paths.